releasenotes/notes/25998.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: release-notes/v2
+kind: feature
+area: security
+issue: 25998
+
+# releaseNotes is a markdown listing of any user facing changes. This will appear in the
```

Comment:
This is not a user facing change

Reply:
so you mean I delete it?

Reply:
Yes, this PR does not need a release note. It is for test analysis by developers

Reply:
removed it